src/collection/dimensions.js

```diff
@@ -925,6 +951,21 @@ defineDimFns( {
   name: 'height'
 } );
 
+elesfn.padding = function(){
+  if(this.length > 0) {
+    var ele = this[0];
+    var _p = ele._private;
+    if( _p.autoPadding !== undefined ) {
```

Comment:
The autopadding could already be defined if changed from a parent node to a non-parent node, so the percent check is still necessary